CMakeLists.txt

```diff
@@ -67,14 +65,30 @@ if(IPDK_TARGET)
 endif()
 
 if(DEPEND_INSTALL_PREFIX)
-  include_directories(${DEPEND_INSTALL_PREFIX}/include)
-  link_directories(${DEPEND_INSTALL_PREFIX}/lib)
+    include_directories(${DEPEND_INSTALL_PREFIX}/include)
+    link_directories(${DEPEND_INSTALL_PREFIX}/lib)
 endif()
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+#####################
+# External packages #
+#####################
+
+if(DEPEND_INSTALL_PREFIX)
+    list(APPEND CMAKE_PREFIX_PATH ${DEPEND_INSTALL_PREFIX})
+endif()
+
+if(WITH_STRATUM)
```

Comment:
Do we need to do similar checks for other dependencies like json, protobuf, gflags, etc

Reply:
Eventually. I figured I'd start with Abseil and do some experimenting before moving on to the others.